src/ts/components/pages/RegisterForm/RegisterForm.tsx

```diff
@@ -8,61 +8,53 @@ import RegisterFormLabels from "src/assets/data/registerForm.json";
 
 type RegisterFormState = {
     /**
-     * first name
+     * The first name of the user who wants to register
      */
-    firstName: string;
+    firstName?: string;
     /**
-     * last name
+     * The last name of the user who wants to register
      */
-    lastName: string;
+    lastName?: string;
     /**
-     * email
+     * The email adress of the user who wants to register
      */
-    email: string;
+    email?: string;
     /**
-     * country
+     * The country the user is living in
      */
-    country: string;
+    country?: string;
```

Comment:
Is there are a reason why they're nullable instead of empty? They're required so there should never be path to an empty string anyway.
This is more convention based than anything, as they have the same meaning, but it should be consistent throughout